### PR TITLE
Add callbacks

### PR DESCRIPTION
**Pull request by thomasthiebaud**
Add simple callback as discussed in #51. Callbacks in this PR are not asynchronous.

### CODE REVIEW

File: src/core/popover.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ export default class Popover extends Element {
       startBtnText: 'Next &rarr;',
       nextBtnText: 'Next &rarr;',
       prevBtnText: '&larr; Previous',
+      onClose: () => {},
```
[the remark this answers is not in the document]
Comment (thomasthiebaud) [reply src/core/popover.js]: Why `0` ? Callbacks are function, so by default I put an empty function.

[the remark this answers is not in the document]
Comment (Yahasana) [reply src/core/popover.js]: execute an empty function is meaningless & slow perf

**Comment by kamranahmedse** [comment]
Thanks for the PR. It has been added in 0.6.0

**Comment by minasvisual** [comment]
I have the "driver.js": "^0.7.1" version and onClose was not implement. when will be implement please?

**Comment by kamranahmedse** [comment]
@minasvisual you should use `onReset`d

**Pull request closed without merging**